pymemcache/client/hash.py

```diff
@@ -261,11 +261,11 @@ def set_many(self, values, *args, **kwargs):
             new_args.insert(0, values)
             result = self._safely_run_func(
                 client,
-                client.set_many, False, *new_args, **kwargs
+                client.set_many, values.keys(), *new_args, **kwargs
```

Comment:
so this means, if any of these keys fail, we mark all as failed?

Reply:
thanks! 
That's true. I will fix it.